bigdl/bigdl.sh

```diff
@@ -66,11 +66,31 @@ if [[ "${ROLE}" == "Master" ]]; then
   # will never come up.
   SPARK_NUM_EXECUTORS=$((SPARK_NUM_EXECUTORS_PER_NODE_MANAGER * CURRENTLY_RUNNING_NODEMANAGERS - 1))
 
-  cat conf/spark-bigdl.conf >> /etc/spark/conf/spark-defaults.conf
+# Check if it BigDL conf or Zoo  
+
+SPARK_BIG_DL_CONF=
```

Comment:
Nit: Consider putting the `cat blah >> blah` in the if statements to avoid this extra variable